plugin-server/tests/helpers/sql.ts

```diff
@@ -207,6 +207,7 @@ export async function createUserTeamAndOrganization(
         test_account_filters: [],
         timezone: 'UTC',
         data_attributes: ['data-attr'],
+        person_display_name_properties: '{email,name,username}',
```

Comment:
Hmm.. this is a bit strange. Would have expected it as an array of strings, much like `app_urls`.

Reply:
Yeah this was me experimenting as a non-empty array ended up with errors only in the plugin-server tests seeding.